source/errors.rst

```diff
@@ -272,7 +272,7 @@ is already used by another user. Compare 8. Slug Already Exists.
 
   {
     "status": 409,
-    "error": "Username already exists",
-    "text": "username ${username} already exists",
+    "error": "Username Already Exists",
+    "text": "Username ${username} already exists",
```

Comment:
The above line also shows that `'username ' + name + ' already exists'` is the correct form.